pyfr/integrators/dual/pseudo/__init__.py

```diff
@@ -1,25 +1,64 @@
 # -*- coding: utf-8 -*-
 
+from pkg_resources import resource_listdir
 import re
 
 from pyfr.integrators.dual.pseudo.pseudocontrollers import BaseDualPseudoController
-from pyfr.integrators.dual.pseudo.pseudosteppers import BaseDualPseudoStepper
+from pyfr.integrators.dual.pseudo.pseudosteppers import BaseDualPseudoStepper, DualDenseRKPseudoStepper
 from pyfr.integrators.dual.pseudo.multip import DualMultiPIntegrator
 from pyfr.util import subclass_where
 
 
+def register_tabulated_pseudo_steppers():
+    # Create subclasses for all tabulated dense schemes
+    rpaths = resource_listdir('pyfr.integrators', 'tabulatedschemes')
+
+    for path in rpaths:
+        m = re.match(r'([a-zA-Z0-9\-~+]+)-s(\d+)(?:-po(\d+))?\.txt$', path)
+        name, nstages, porder = m.group(1), m.group(2), m.group(3)
+
+        attrs = dict(pseudo_stepper_name=name, nstages=int(nstages),
+                     path=path)
+
+        if porder:
+            attrs['pseudo_stepper_porder'] = int(porder)
+            name += '_po{}'.format(porder)
+
+        register_tabulated_pseudo_steppers.__setattr__(
```

Comment:
Probably worth explaining what you're doing here.  Any reason we can't use setattr here (the ordinary function)?

Reply:
Also, why not set register_tabulated_pseudo_steppers._schemes = [] and then .append to the list in this loop?